turbo/src/main/kotlin/dev/hotwire/turbo/views/TurboView.kt

```diff
@@ -36,6 +34,13 @@ class TurboView @JvmOverloads constructor(context: Context, attrs: AttributeSet?
             return
         }
 
+        webView.updateLayoutParams {
+            height = when (webViewContainer) {
+                is ScrollingView -> LayoutParams.WRAP_CONTENT
+                else -> LayoutParams.MATCH_PARENT
+            }
+        }
```

Comment:
Does this work if you move this down to line 55? It feels safer to me to only update the layout params if we know that we're attaching the `webView` to a new parent.